src/java.base/share/classes/com/sun/crypto/provider/CipherBlockChaining.java

```diff
@@ -148,8 +153,15 @@ int encrypt(byte[] plain, int plainOffset, int plainLen,
         ArrayUtil.blockSizeCheck(plainLen, blockSize);
         ArrayUtil.nullAndBoundsCheck(plain, plainOffset, plainLen);
         ArrayUtil.nullAndBoundsCheck(cipher, cipherOffset, plainLen);
-        return implEncrypt(plain, plainOffset, plainLen,
-                           cipher, cipherOffset);
+        int processed = 0;
+        for (;  plainLen > chunkSize; cipherOffset += chunkSize,
+            plainOffset += chunkSize, plainLen -= chunkSize) {
+            processed +=
+               implEncrypt(plain, plainOffset, chunkSize, cipher, cipherOffset);
+        }
+        processed +=
+            implEncrypt(plain, plainOffset, plainLen, cipher, cipherOffset);
```

Comment:
Why do we have plainLen > chunkSize and not plainLen >= chunkSize?  A small comment on that would be helpful.